tests/specs/sass-spec-exclude-warning.txt

```diff
@@ -81,6 +71,7 @@ libsass-closed-issues/issue_308
 libsass-closed-issues/issue_674
 libsass-closed-issues/issue_759
 libsass-todo-issues/issue_1096
+libsass-todo-issues/issue_1798/3
```

Comment:
Those 2 added tests are tests that trigger the deprecation warning `DEPRECATION WARNING: Unterminated interpolations in multiline comments are deprecated and will be removed in ScssPhp 2.0`.
Those are false positives of the warning, because those comments are not preserved in the scss output (and so interpolation will not be parsed in them)

Reply:
and now fixed, as I found the issue